src/__experimental__/components/input-icon-toggle/input-icon-toggle.component.js

```diff
@@ -12,9 +13,17 @@ const InputIconToggle = ({
 }) => {
   if (disabled || readOnly) return null;
 
+  const inputContext = useContext(InputPresentationContext);
+
   return (
-    <InputIconToggleStyle key='label-icon' { ...props }>
-      { children || <Icon type={ type } /> }
+    <InputIconToggleStyle
+      key='label-icon'
+      inputFocus={ inputContext.hasFocus }
+      inputHover={ inputContext.hasHover }
```

Comment:
hover and focus should be also triggered on a label component

Reply:
Thanks Kuba, totally missed that! 👍

Reply:
Re-worked now to apply the correct updates on Label hover / focus as well